jenkins/deduper.py

```diff
@@ -20,6 +21,7 @@
 
 
 class GhprbOutdatedBuildAborter:
+
```

Comment:
Not the plugin, but I ran the repo through autopep8.
Is it annoying? There were a lot of pep8 violations, and it was the easiest way to clean them up. :)